cartography/intel/aws/ec2/tgw.py

```diff
@@ -0,0 +1,214 @@
+import logging
+
+from .util import get_botocore_config
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_transit_gateways(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    return client.describe_transit_gateways()
+
+
+@timeit
+def get_tgw_attachments(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    paginator = client.get_paginator('describe_transit_gateway_attachments')
+    tgw_attachments = []
+    for page in paginator.paginate():
+        tgw_attachments.extend(page['TransitGatewayAttachments'])
+    return {"TransitGatewayAttachments": tgw_attachments}
+
+
+@timeit
+def get_tgw_vpc_attachments(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    paginator = client.get_paginator('describe_transit_gateway_vpc_attachments')
+    tgw_vpc_attachments = []
+    for page in paginator.paginate():
+        tgw_vpc_attachments.extend(page['TransitGatewayVpcAttachments'])
+    return {"TransitGatewayVpcAttachments": tgw_vpc_attachments}
+
+
+@timeit
+def load_transit_gateways(neo4j_session, data, region, current_aws_account_id, aws_update_tag):
```

Comment:
Please add integration tests with dummy data so that we know the new nodes are created with the expected relationships. You can copy https://github.com/lyft/cartography/blob/master/tests/integration/cartography/intel/aws/test_eks.py.

Reply:
Added some basic tests just checking that TGWs and TGW attachments are added.